plugins/Dashboard/tests/UI/WidgetizedDashboard_spec.js

```diff
@@ -0,0 +1,334 @@
+        await page.evaluate(() => $('#newDashboardName').val('newname'));
+        await page.waitForTimeout(250);
+        var button = await page.jQuery('.modal.open .modal-footer a:contains(Save)');
+        await button.click();
+        await page.mouse.move(-10, -10);
+        await page.waitForNetworkIdle();
+
+        expect(await page.screenshot({ fullPage: true })).to.matchImage('rename');
+    });
+
+    it("should copy dashboard successfully when copy dashboard process completed", async function() {
+        await page.click('.dashboard-manager .title');
+        await page.click('li[data-action="copyDashboardToUser"]');
+        await page.waitForTimeout(100); // wait for animation
+        await page.evaluate(function () {
+            $('#copyDashboardName').val('');
+        });
+        await page.type('#copyDashboardName', 'new <dash> 💩');
+        await page.waitForSelector('#copyDashboardUser [value="superUserLogin"]');
+        await page.select('#copyDashboardUser', 'superUserLogin');
+        var button = await page.jQuery('.modal.open .modal-footer a:contains(Ok)');
+        await button.click();
+        await page.waitForFunction("$('.ui-confirm :contains(\"Current dashboard successfully copied to selected user.\").length > 0')");
+
+        await page.goto(url.replace("idDashboard=1", "idDashboard=6"));
+        await page.mouse.move(-10, -10);
+        await page.waitForNetworkIdle();
+
+        expect(await page.screenshot({ fullPage: true })).to.matchImage('copied');
+    });
+
+    it("should reset dashboard when reset dashboard process completed", async function() {
+        await page.click('.dashboard-manager .title');
+        await page.click('li[data-action="resetDashboard"]');
+        await page.waitForSelector('.modal.open');
+        var button = await page.jQuery('.modal.open .modal-footer a:contains(Yes)');
+        await button.click();
+        await page.waitForNetworkIdle();
+        await page.mouse.move(-10, -10);
+        await page.waitForSelector('.widget');
+        await page.waitForNetworkIdle();
+        await page.waitForTimeout(500); // wait for widgets to render fully
+
+        expect(await page.screenshot({ fullPage: true })).to.matchImage('reset');
+    });
+
+    it("should remove dashboard when remove dashboard process completed", async function() {
+        await page.click('.dashboard-manager .title');
+        await page.click('li[data-action="removeDashboard"]');
+        await page.waitForSelector('.modal.open');
+        var button = await page.jQuery('.modal.open .modal-footer a:contains(Yes)');
+        await button.click();
+        await page.mouse.move(-10, -10);
+        await page.waitForTimeout(200);
+        await page.waitForNetworkIdle();
+        await page.waitForSelector('.widget');
+        await page.waitForNetworkIdle();
+
+        expect(await page.screenshot({ fullPage: true })).to.matchImage('removed');
```

Comment:
This one consistently fails for me locally :thinking: 

With the change I locally get `newname` selected in first position, as I would actually expect (was at that position in both `reset` and `copied`, as per the expected screenshots).

For the expected screenshot the `newname` magically moves to the last position in the `removed` screenshot, being first place before that.  Interesting there is no CI failure and screenshot diff for that :monocle_face:

Reply:
That's really strange. The screenshot needs an update, but the test somehow doesn't fail in the action. The image processed in the action and the expected one clearly differ. Running the `compare` command on them, like the test does also returns that they differ in width or height, so there needs to be another weird reason to let the test pass nevertheless.